test/elements-test.js

```diff
@@ -1829,6 +1829,14 @@ describe("elements", () => {
       buttons[0].removeEventListener("click", increment);
       buttons[0].click();
       expect(clickCount).to.equal(2);
+
+      let focusCount = 0;
+      buttons[0].addEventListener("focus", () => {
```

Comment:
I would split these tests up: "listens to click event", "listens to focus event" for more clarity when something breaks.

Reply:
Good point, as usual. Splitting ✅